From 99a9a51f7ae3d8126d40c3bcc16a223f3f4f50b4 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Mon, 24 Oct 2022 11:54:12 +0200
Subject: [PATCH 0734/1695] plat-stm32mp1: add ahb2 bus support

Maps AHB2 bus IOMEM for accessing STM32MP13 ADC1&2 peripherals.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git f64f23bb4beb92ee618673ce293dce4b205e1b33 plat-stm32mp1: add ahb2 bus support]
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Iab4d01f89a16f3106dec3cf17e931c179e2bf000
---
 core/arch/arm/plat-stm32mp1/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp1/main.c b/core/arch/arm/plat-stm32mp1/main.c
index 07c007e59..3c6268e85 100644
--- a/core/arch/arm/plat-stm32mp1/main.c
+++ b/core/arch/arm/plat-stm32mp1/main.c
@@ -54,6 +54,7 @@ register_phys_mem_pgdir(MEM_AREA_IO_SEC, APB5_BASE, APB5_SIZE);
 #ifdef CFG_STM32MP13
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, APB6_BASE, APB6_SIZE);
 #endif
+register_phys_mem_pgdir(MEM_AREA_IO_SEC, AHB2_BASE, AHB2_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, AHB4_BASE, AHB4_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, AHB5_BASE, AHB5_SIZE);
 register_phys_mem_pgdir(MEM_AREA_IO_SEC, GIC_BASE, GIC_SIZE);
-- 
2.47.2

